modules/heat_conduction/include/interfacekernels/ConjugateHeatTransfer.h

```diff
@@ -0,0 +1,35 @@
+#include "InterfaceKernel.h"
+
+class ConjugateHeatTransfer;
```

Comment:
Get rid of that dirty forward declaration that we don't need anymore!
```suggestion
```

Reply:
Thanks for the review! This was pulled over from Pronghorn almost exactly and then updated, so I see I've missed a few things. I was surprised it wasn't already in too - though I suppose it was one of those "we need it now, so throw it into the app and we can come back to the module later" things.